src/App.js

```diff
@@ -0,0 +1,65 @@
+import React, { Fragment, useState, useEffect } from 'react';
+import { isExpired, decodeToken } from 'react-jwt';
```

Comment:
ah that's handy :)

Reply:
It isn't a library I've used before, but it seems good. Is pretty simple code